packages/executable/helpers/pkhCopyCrossPlatforms.js

```diff
@@ -98,8 +105,10 @@ Promise.all(promises).then(() => {
     let outputPath = `${zipPath}/SERGE_${getDate()}`
     if (dir.indexOf('linux') !== -1) {
       outputPath += '_linux.zip'
-    } else if (dir.indexOf('macos') !== -1) {
+    } else if (dir.indexOf('macos') !== -1 && dir.indexOf('macos-arm64') === -1) {
```

Comment:
@lebaphi  - the logic can be simplified if you change the order a little.

If you test for arm64 first, you just need to test for `macos` in the following test